test/integration/volume/attach_detach_test.go

```diff
@@ -361,6 +549,69 @@ func TestPodUpdateWithKeepTerminatedPodVolumes(t *testing.T) {
 	waitForPodFuncInDSWP(t, ctrl.GetDesiredStateOfWorld(), 20*time.Second, "expected non-zero pods in dsw if KeepTerminatedPodVolumesAnnotation is set", 1)
 }
 
+func waitForMetric(t *testing.T, m basemetric.CounterMetric, expectedCount float64, identifier string) {
+	if err := wait.Poll(100*time.Millisecond, 60*time.Second, func() (bool, error) {
+		gotCount, err := metricstestutil.GetCounterMetricValue(m)
+		fmt.Println(gotCount)
```

Comment:
or this slipped from your local test or use t.Logf with a more meaninful message so it is debuggable